libs/partners/anthropic/langchain_anthropic/chat_models.py

```diff
@@ -196,7 +196,22 @@ def _format_messages(messages: List[BaseMessage]) -> Tuple[Optional[str], List[D
                         f"Content items must be str or dict, instead was: {type(item)}"
                     )
         else:
-            content = message.content
+            if (
+                isinstance(message, AIMessage)
+                and message.content == ""
+                and message.tool_calls
+            ):
+                content = [
```

Comment:
can we support putting string content, if not empty, as an additional part of type text?